src/Symfony/Component/Security/Core/Resources/translations/security.mn.xlf

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="mn" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Нэвтрэх хүсэлтийн алдаа гарав.</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Нэвтрэх эрхийн мэдээлэл олдсонгүй.</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <target>Системийн алдаанаас болон нэвтрэх хүсэлтийг гүйцэтгэх боломжгүй байна.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>Invalid credentials.</source>
+                <target>Буруу нэвтрэх эрхийн мэдээлэл.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>Cookie has already been used by someone else.</source>
+                <target>Күүки файлыг аль хэдийн өөр хүн хэрэглэж байна.</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>Not privileged to request the resource.</source>
+                <target>Энэхүү мэдээллийг өгөхөд эрх хүрэхгүй байна.</target>
+            </trans-unit>
+            <trans-unit id="7">
+                <source>Invalid CSRF token.</source>
+                <target>Буруу CSRF токен.</target>
```

Comment:
Google translates this to "Reverse CSRF token". =)

Reply:
yeah there is no direct translation for CSRF since it is a technical word but the invalid part is correct